Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM node:20.11-bookworm-slim AS node-image
-FROM python:3.12.7-slim-bookworm
+FROM node:20.18-bookworm-slim AS node-image
+FROM python:3.12.8-slim-bookworm
+# https://docs.astral.sh/uv/guides/integration/docker/#installing-uv
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
 
 # Requirements for building packages
-RUN apt-get update \
+RUN uv venv \
```

Comment:
What's `uv venv` do here? Feels like it might fit in better with `RUN pip3` on line 33 instead of with apt?